config/config.default.yaml

```diff
@@ -1072,6 +1129,9 @@ data:
     source: "build"
     version_or_latest: "latest"
 
+secrets:
+  corine: '' #Add API key here if primary source is used for retrieving corine dataset after registering in https://land.copernicus.eu/user/login
```

Comment:
Please document this like here: https://github.com/PyPSA/pypsa-eur/blob/b3d7c3301e6855efd081829406be79d4e673f387/doc/configtables/toplevel.csv?plain=1#L10-L11